### PR TITLE
CEM-1074 turning Image story from js to tsx

### PR DESCRIPTION
**Pull request by williancozzi**
Conversion of Image from .js to .tsx

image

### CODE REVIEW

File: stories/Inputs/Image.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+export default {
+    title: createStoryTitle('Image'),
+    component: Image,
+} as Meta;
+
+export const Basic: Story<ImageProps> = () => {
+    const [src, setSrc] = useState('');
+    
+    function addImage(data) {
```
Comment (ralph-dev) [inline stories/Inputs/Image.stories.tsx:14]: Use ES6 format
Const

File: stories/Inputs/Image.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React, { useState } from 'react';
+import { Meta, Story } from '@storybook/react';
+import {createStoryTitle} from "../Constants";
+import { Image, ImageProps } from '../../src';
+
+export default {
+    title: createStoryTitle('Image'),
+    component: Image,
```
Comment (ralph-dev) [inline stories/Inputs/Image.stories.tsx:8]: Shouldnt the add image be in arg types

**Review by ralph-dev — CHANGES_REQUESTED** [review]

File: stories/Inputs/Image.stories.tsx (hunk 1 above)
Comment (williancozzi) [reply stories/Inputs/Image.stories.tsx]: Changed to ES6

**Review by ralph-dev — APPROVED** [review]
LGTM

**Pull request merged**